### PR TITLE
[new release] xen-gnt (2 packages) (4.0.2)

### PR DESCRIPTION
**Pull request by hannesm**
Xen grant table bindings for OCaml

- Project page: <a href="https://github.com/mirage/ocaml-gnt">https://github.com/mirage/ocaml-gnt</a>
- Documentation: <a href="https://mirage.github.io/ocaml-gnt/">https://mirage.github.io/ocaml-gnt/</a>

##### CHANGES:

- remove mirage-profile from dependency tree (mirage/ocaml-gnt#47, by @hannesm)

### CODE REVIEW

**Comment by hannesm** [comment]
I reported the C compiler issues upstream at https://github.com/mirage/ocaml-gnt/issues/48

**Comment by hannesm** [comment]
CI failures - revdeps is nothing new (all unrelated, some solver timeouts etc.)

oraclelinux: here conf-xen tries to install xen-devel which is not available. this to be addressed in a separate PR
fedora: C compiler warnings & errors: reported upstream at https://github.com/mirage/ocaml-gnt/issues/48

TL;DR: this PR is good to be merged!

**Comment by avsm** [comment]
Thanks for the triage!
